app/src/main/res/layout/activity_login.xml

```diff
@@ -5,68 +5,140 @@
+        android:orientation="vertical"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
+
+        <android.support.design.widget.TextInputLayout
+            style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox"
+            android:layout_width="300dp"
+            android:layout_height="60dp">
+
+            <android.support.design.widget.TextInputEditText
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:hint="Username"
+                android:maxLines="1"
+                android:textSize="20sp" />
+        </android.support.design.widget.TextInputLayout>
+
+        <android.support.design.widget.TextInputLayout
+            style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox"
+            android:layout_width="300dp"
+            android:layout_height="60dp"
+            android:layout_marginTop="10dp">
+
+            <android.support.design.widget.TextInputEditText
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:hint="Password"
+                android:inputType="text"
+                android:maxLines="1"
+                android:textSize="20sp" />
+        </android.support.design.widget.TextInputLayout>
+
+    </LinearLayout>
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignTop="@+id/upper_poly"
+        android:layout_alignEnd="@+id/upper_poly"
+        android:layout_marginStart="230dp"
+        android:layout_marginTop="420dp"
+        android:text="Forgot?"
+        android:textSize="20sp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <Button
+        android:layout_width="150dp"
+        android:layout_height="45dp"
+        android:layout_alignStart="@+id/upper_poly"
+        android:layout_alignTop="@+id/upper_poly"
+        android:layout_alignEnd="@+id/upper_poly"
+        android:layout_marginStart="200dp"
+        android:layout_marginTop="460dp"
+        android:background="@drawable/button_signin"
+        android:text="Sign in"
+        android:textColor="#FFFFFF"
+        android:textSize="25sp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <ImageButton
+        android:layout_width="100dp"
+        android:layout_height="100dp"
+        android:layout_alignStart="@+id/upper_poly"
+        android:layout_alignTop="@+id/upper_poly"
+        android:layout_marginStart="50dp"
+        android:layout_marginTop="510dp"
+        android:background="@color/transparent"
+        android:src="@drawable/ic_facebook"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
```

Comment:
The attributes of these two Image Button should be in styles.xml, they are nearly common. Secondly, change it to ImageView and then add OnClickListener